vmdb/app/models/ems_refresh/parsers/openshift.rb

```diff
@@ -14,17 +14,16 @@ def get_routes(inventory)
       end
 
       def get_projects(inventory)
-        process_collection(inventory["project"], :container_projects) { |n| parse_project(n) }
+        inventory["project"].each { |item| parse_project(item) }
```

Comment:
cc @Fryguy this is correct from our point of view, I am not sure if that's ok with you.